### PR TITLE
Fibonacci_XOR problem added

### PR DESCRIPTION
**Pull request by riyavaryani**

### CODE REVIEW

**Comment by GauravWalia19** [comment]
@riyavaryani please raise a issue with the HacktoberFest20 template.

File: MISC/Fibonacci_xor/C++/fibonacci_xor.cpp [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+/*You are given three integers a, b, and n, calculate f(n), where f(n) is the xor of last two numbers
+i.e, f(n) = f(n-1)?f(n-2), where ? is xor operation.
+*/
+
+
+#include<bits/stdc++.h>
+using namespace std;
+int main()
+{
+  int a,b,n;
```
Comment (GauravWalia19) [inline MISC/Fibonacci_xor/C++/fibonacci_xor.cpp:11]: Here please use good naming like

int firstNumber, secondNumber, n;

**Review by GauravWalia19 — CHANGES_REQUESTED** [review]
Please update the cpp file with the mentioned comments
And also update the README file

**Comment by riyavaryani** [comment]
changes done

**Review by GauravWalia19 — APPROVED** [review]

**Pull request merged**